### PR TITLE
Add timers to movement Tasks.

### PR DESCRIPTION
**Pull request by LReinel**
Add timers and finish signals (sound and vibration) to Movement tasks (Free walking, balance and postural tremor).

### CODE REVIEW

File: app/src/main/assets/instructions.csv [hunk 1]
```diff
@@ -29,8 +29,8 @@
...
-31;Movement;gait;Gait;Walk a short path;Please walk from your bedroom to your kitchen and come back during **four** times, Press start button when you are ready.;Gehen;Gehen deutsch beschreibung;Gehen deutsch anweisung;Marcha;Recorrrer un camino corto;Manteniendo el celular en su bolsillo, recorra un camino corto de ida y vuelta 4 veces. Por ejemplo, desde su habitación hasta la cocina. Cuando este listo, presione el botón Empezar.;
-32;Movement;gait;Gait-Free;Walk around;Please walk around your house during **two minutes**. Press start button when you are ready.;Gehen 2;Gehen2 deutsch beschreibung;Gehen2 deutsch anweisung;Marcha libre;Caminar libremente;Camine libremente durante 2 minutos. Cuando esté listo, presione el botón Empezar.;
+31;Movement;gait1;Gait;Walk a short path;Please walk from your bedroom to your kitchen and come back during **four** times, Press start button when you are ready.;Gehen;Gehen deutsch beschreibung;Gehen deutsch anweisung;Marcha;Recorrrer un camino corto;Manteniendo el celular en su bolsillo, recorra un camino corto de ida y vuelta 4 veces. Por ejemplo, desde su habitación hasta la cocina. Cuando este listo, presione el botón Empezar.;
```
Comment (jcvasquezc) [inline app/src/main/assets/instructions.csv]: Please use more reliable names related t the name of the tasks
gait1 and gait2 does not mean anything

File: app/src/main/java/com/sma2/sma2/DataAccess/ExerciseSessionManager.java [hunk 2]
```diff
@@ -296,7 +297,7 @@ else if (ExFrag.equals("postural"))
                         Uri.parse("Instruction/Path"),
                         Ex_postural_Rec.class));
             }
-            else if (ExFrag.equals("gait"))
+            else if (ExFrag.equals("gait1"))
```
Comment (jcvasquezc) [inline app/src/main/java/com/sma2/sma2/DataAccess/ExerciseSessionManager.java]: change the name here properly according to my first comment

File: app/src/main/java/com/sma2/sma2/DataAccess/ExerciseSessionManager.java [hunk 3]
```diff
@@ -308,6 +309,18 @@ else if (ExFrag.equals("gait"))
                         Uri.parse("Instruction/Path"),
                         Ex_Walking_Rec.class));
             }
+            else if (ExFrag.equals("gait2"))
```
Comment (jcvasquezc) [inline app/src/main/java/com/sma2/sma2/DataAccess/ExerciseSessionManager.java]: change gait2 with the proper name

File: app/src/main/java/com/sma2/sma2/ExerciseFragments/Ex_postural_Rec.java [hunk 4]
```diff
@@ -17,8 +17,8 @@
 
 public class Ex_postural_Rec extends ExerciseFragment implements ButtonFragment.OnButtonInteractionListener {
     private MovementRecorder recorder;
-    private static long START_COUNTDOWN = 3;
-    private static long EXERCISE_TIME = 30;
+    private static long START_COUNTDOWN = 5;
+    private static long EXERCISE_TIME = 10;
```
Comment (jcvasquezc) [inline app/src/main/java/com/sma2/sma2/ExerciseFragments/Ex_postural_Rec.java]: why did you change this?

Comment (LReinel) [reply app/src/main/java/com/sma2/sma2/ExerciseFragments/Ex_postural_Rec.java]: Postural tasks are of ten seconds
